src/Bech32Address.h

```diff
@@ -0,0 +1,67 @@
+// Copyright © 2017-2019 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include "Data.h"
+#include "PublicKey.h"
+#include <TrezorCrypto/hasher.h>
+
+#include <string>
+#include <memory>
+
+namespace TW {
+
+/// Bech32 string address, used by Cosmos, etc.  Variation are the hash function.
+class Bech32Address {
+private:
+    /// Chain-specific human-readable part.  If missing (""), no prefix check is done.
+    /// \see https://github.com/satoshilabs/slips/blob/master/slip-0173.md
+    std::string _hrp;
```

Comment:
Existing code doesn't have `_` prefix for private members

Reply:
ok, accepted